backend/api/Controllers/DeckController.cs

```diff
@@ -141,6 +144,53 @@ public async Task<ActionResult<Deck>> Create([FromBody] CreateDeckQuery deck)
             }
         }
 
+        /// <summary>
+        /// Set default localization are for a deck
+        /// </summary>
+        /// <remarks>
+        /// <para> This query sets the default localization area for a deck </para>
+        /// </remarks>
+        [HttpPost]
+        [Authorize(Roles = Role.Admin)]
+        [Route("{deckId}/{areaId}/set-default-localization-area")]
+        [ProducesResponseType(typeof(AreaResponse), StatusCodes.Status201Created)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        public async Task<ActionResult<Deck>> SetDefaultLocalizationArea(
+            [FromRoute] string deckId,
+            [FromRoute] string areaId
+        )
+        {
+            _logger.LogInformation(@"Setting default localization area {AreaId} to deck {DeckId}", areaId, deckId);
+            var deck = await _deckService.ReadById(deckId);
+            if (deck == null)
+                return NotFound($"Could not find deck with id {deckId}");
+
+            var area = await _areaService.ReadById(areaId);
+            if (area == null)
+                return NotFound($"Could not find area with id {areaId}");
+
+            if (area.Deck == null)
+                return Forbid($"Area {areaId} is not linked to any deck");
```

Comment:
```suggestion
                return NotFound($"Area {areaId} is not linked to any deck");
```
Assuming `Forbid` means 403 Forbidde that should not be used here as it indicates that the user does not have access to the requested resource which is not correct.